datadog_checks_base/tests/test_prometheus.py

```diff
@@ -1601,3 +1601,30 @@ def test_health_service_check_failing():
         PrometheusCheck.CRITICAL,
         tags=["endpoint:http://fake.endpoint:10055/metrics"]
     )
+
+def test_set_prometheus_timeout():
```

Comment:
👍 Added some tests relative to that, both for `PrometheusCheck` and `GenericPrometheusCheck`

Reply:
Ah, since we're removing the validation part, we don't need to check for negative values, just that the same value that we set is the same as what we get.

Reply:
Sure, I was a bit confused by your two remarks, so I did both ^^